static/resources.json

```diff
@@ -155,6 +139,144 @@
       }
     }
   },
+  "Books": {
+    "description": "The best books for learning Python or Python Frameworks",
+    "resources": {
+      "Automate the Boring Stuff with Python": {
+        "description": "One of the best books out there for Python beginners. You can buy a copy, but there's also a free online version.",
+        "urls": [
+          {
+            "title": "Website",
+            "url": "https://automatetheboringstuff.com/",
+            "icon": "regular/link"
+          },
+          {
+            "title": "Amazon",
+            "url": "http://www.amazon.com/gp/product/1593275994/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=1593275994&linkCode=as2&tag=playwithpyth-20&linkId=HDM7V3T6RHC5VVN4",
+            "icon": "branding/amazon"
+          }
+        ],
+        "payment": "optional",
+        "payment_description": "A free version is available online, with the option to buy a physical copy"
+      },
+      "Fluent Python": {
+        "description": "Clear, Concise, and Effective Programming",
+        "urls": [
+          {
+            "title": "Website",
+            "url": "https://www.safaribooksonline.com/library/view/fluent-python/9781491946237/",
+            "icon": "regular/link"
+          },
+          {
+            "title": "Amazon",
+            "url": "https://www.amazon.com/Fluent-Python-Concise-Effective-Programming/dp/1491946008",
+            "icon": "branding/amazon"
+          },
+          {
+            "title": "GitHub",
+            "url": "https://github.com/fluentpython",
+            "icon": "branding/github"
+          }
+        ],
+        "payment": "paid"
+      },
+      "Effective Python": {
+        "description": "59 Specific Ways to Write Better Python",
+        "urls": [
+          {
+            "title": "Website",
+            "url": "https://effectivepython.com/",
+            "icon": "regular/link"
+          },
+          {
+            "title": "Amazon",
+            "url": "https://www.amazon.com/Effective-Python-Specific-Software-Development/dp/0134034287",
+            "icon": "branding/amazon"
+          },
+          {
+            "title": "GitHub",
+            "url": "https://github.com/bslatkin/effectivepython",
+            "icon": "branding/github"
+          }
+        ],
+        "payment": "paid"
+      },
+      "A Byte of Python": {
+        "description": "A free book on programming using the Python language. It serves as a tutorial or guide to the Python language for a beginner audience. If all you know about computers is how to save text files, then this is the book for you.",
+        "urls": [
```

Comment:
There is a physical version that can be bought at https://www.swaroopch.com/buybook/